plugin/fiftyone/app/models/schemas.py

```diff
@@ -36,6 +39,18 @@ class Task(BaseModel):
     tid: str
     datas: List[DataSet]
 
+    @validator("datas")
+    def check_ground_truth_once(cls, v):
+        count = 0
```

Comment:
from collections import Counter

type_cnt = Counter(d.data_type for d in v)
if type_cnt[DataSetResultTypes.GROUND_TRUTH] > 1:
    raise xxx